pkg/proxy/proxy.go

```diff
@@ -149,6 +168,78 @@ func (p *Proxy) StartProxy() *http.Server {
 	return srv
 }
 
+// unsecured returns true if the request does not require authentication
+func unsecured(ctx echo.Context) bool {
+	uri := ctx.Request().URL.RequestURI()
+	return uri == proxyHealthEndpoint || uri == "/.well-known/oauth-authorization-server" || strings.HasPrefix(uri, authEndpoint)
```

Comment:
it's just a detail, but could you please (for easy navigation) store the `/.well-known/oauth-authorization-server` path as a constant and use it also in 
```
router.Any("/.well-known/oauth-authorization-server", p.oauthConfiguration)
```

Reply:
Done in https://github.com/codeready-toolchain/registration-service/pull/382/commits/5a8e586828d7e638e5cc85c94d28e8dc1e25302c